<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="qtViewer3d" name="logic_View" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="start">
    <property name="title">
     <string>开始</string>
    </property>
    <widget class="QMenu" name="menuopen">
     <property name="title">
      <string>open</string>
     </property>
     <addaction name="actionstep"/>
    </widget>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionxml"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuopen"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>建模</string>
    </property>
    <widget class="QMenu" name="menuPrimAPI">
     <property name="title">
      <string>PrimAPI</string>
     </property>
     <addaction name="actionBox"/>
    </widget>
    <addaction name="menuPrimAPI"/>
   </widget>
   <addaction name="start"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="object_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>construct_and_change</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="Logic_ConstructView" name="logic_Construct"/>
  </widget>
  <widget class="QDockWidget" name="view_message_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>可视化信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="Logic_ScreenData" name="logic_ViewData"/>
  </widget>
  <widget class="QDockWidget" name="document_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>document</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Logic_DocView" name="logic_DocData"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionstep">
   <property name="text">
    <string>step</string>
   </property>
  </action>
  <action name="actionxml">
   <property name="text">
    <string>xml</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="text">
    <string>Box</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtViewer3d</class>
   <extends>QWidget</extends>
   <header>widgets.Logic_Viewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Logic_DocView</class>
   <extends>QWidget</extends>
   <header>widgets.Logic_DocView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Logic_ConstructView</class>
   <extends>QWidget</extends>
   <header>widgets.Logic_ConstructView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Logic_ScreenData</class>
   <extends>QWidget</extends>
   <header>widgets.Logic_ScreenData</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
